src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -1,11 +1,44 @@
 package core.basesyntax.service;
 
+import core.basesyntax.dao.StorageDaoImpl;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MIN_PASSWORD_LENGTH = 6;
+    private static final int MIN_AGE = 18;
+    private final StorageDaoImpl storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        correctUserData(user);
+        correctUserAge(user);
+        correctLogin(user);
+        correctPassword(user);
```

Comment:
```suggestion
        validateUserData(user);
        validateUserAge(user);
        validateLogin(user);
        validatePassword(user);
```
Names of methods must be verbs